recipe/meta.yaml

```
@@ -18,45 +18,46 @@ build:
 requirements:
   host:
     - python
-    - flit-core >=3.2,<4
     - pip
+    - flit-core >=3.2,<4
   run:
     - python
 
 test:
   requires:
     - pip
-    - aiohttp            # [not (linux and s390x)]
-    - pyopenssl          # [not (linux and s390x)]
-    - pytest             # [not (linux and s390x)]
-    - pytest-asyncio     # [not (linux and s390x)]
-    - pytest-httpserver  # [not (linux and s390x)]
-    - urllib3            # [not (linux and s390x)]
-    - trustme            # [not (linux and s390x)]
-    - requests           # [not (linux and s390x)]
-    - flaky              # [not (linux and s390x)]
-    - httpx              # [not (linux and s390x)]
+    - aiohttp            # [not linux]
+    - pyopenssl          # [not linux]
+    - pytest             # [not linux]
+    - pytest-asyncio     # [not linux]
+    - pytest-httpserver  # [not linux]
+    - urllib3            # [not linux]
+    - trustme            # [not linux]
+    - requests           # [not linux]
+    - flaky             # [not linux]
+    - httpx             # [not linux]
```

```suggestion
    - aiohttp
    - pyopenssl
    - pytest
    - pytest-asyncio
    - pytest-httpserver
    - urllib3
    - trustme
    - requests
    - flaky
    - httpx
```
Run tests on all platforms